Vixen.System/Commands/Named8BitCommand.cs

```diff
@@ -0,0 +1,65 @@
+﻿namespace Vixen.Commands
+{
+    /// <summary>
+    /// Maintains a named 8 bit command.
+    /// </summary>
+    /// <typeparam name="T">Type of index command</typeparam>
+    public class Named8BitCommand<T> : _8BitCommand
+		where T : System.Enum
+	{
+		#region Constructors 
+
+		public Named8BitCommand(byte value) :
+			base(value)
+		{
+		}
+
+		public Named8BitCommand(short value) : 
+			this((byte)value)
+		{
+		}
+
+		public Named8BitCommand(int value) : 
+			this((byte)value)
+		{
+		}
+
+		public Named8BitCommand(long value)
+			: this((byte)value)
+		{
+		}
+
+		public Named8BitCommand(float value)
+			: this((byte)value)
+		{
+		}
+
+		public Named8BitCommand(double value)
+			: this((byte)value)
+		{
+		}
+
+        #endregion
+
+        #region Public Properties
+
+		/// <summary>
+		/// Type of index command value.  This property allows the preview to recognize commands.
```

Comment:
Do we need to mention preview in the comment here? Can this be referred to more generically since there could be other consumers than the preview that might use this information.

Reply:
I attempted to address this in the last version I pushed.  I guess let me know if you want me to remove the remark comments as they are fixture specific.  I know I add a lot more comments than is typical in the Vixen code base but it is somewhat part of my process.  I try to review my own code before submitting it and part of that process is adding documentation and describing the logic flow in comments.

Reply:
I am all for comments. I think in this case, even though this was created for the fixtures, that may not be the only possible use of it in the future. Let's not lock it into an implementation as these should be more universal. I think the property name change to Label is fine as it provides a more universal name for the fields purpose. The same applies to the comments for those fields. Instead of mentioning preview or intelligent fixtures, describe the label in more generic terms. Something like: Provides a customizable descriptive string field that labels or describes the intent of the command. 

You can also be more specific in the details of the commit message if you want o convey this particular implementation of it. This is where you can put that the Label is used by the Preview to provide functionality. This documents the more specific implementation without locking it to the code that can be more universal.

Reply:
Updated comments as suggested.